flink-connector-aws/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/FlinkKinesisConsumer.java

```diff
@@ -149,6 +149,7 @@ public class FlinkKinesisConsumer<T> extends RichParallelSourceFunction<T>
             sequenceNumsToRestore;
 
     private volatile boolean running = true;
+    private volatile boolean closed = false;
```

Comment:
Can we add a doc to explain difference between `running` and `closed`?

Reply:
Added comments for these flags